src/SourceDialog/SourceDialog.vala

```diff
@@ -139,6 +141,11 @@ public class Maya.View.SourceDialog : Gtk.Grid {
         color_button_green_context.add_class ("green");
         color_button_green_context.add_provider (css_provider, Gtk.STYLE_PROVIDER_PRIORITY_APPLICATION);
 
```

Comment:
I think you forgot to create the new color button `color_button_mint = new Gtk.RadioButton.from_widget (color_button_red);`

Reply:
So true!

Reply:
Oh yes, after I wrote this line a few times before, it actually slipped through my fingers. :smile: